api/test/e2e/oidc/oidc_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package oidc_test
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+
+	"github.com/apisix/manager-api/test/e2e/base"
+)
+
+var _ = ginkgo.Describe("Oidc", func() {
+
+	table.DescribeTable("test oidc module",
+		func(tc base.HttpTestCase) {
+			base.RunTestCase(tc)
+		},
+
+		// apisix/admin/oidc/login
+		table.Entry("Access the oidc/login", base.HttpTestCase{
+			Object:       base.ManagerApiExpect(),
+			Method:       http.MethodGet,
+			Path:         "/apisix/admin/oidc/login",
+			ExpectStatus: http.StatusOK,
```

Comment:
Why the status is `http.StatusOK `? I guess it should be 302.

Reply:
And I think you should save the cookie， and request with the cookie in the later request.